### DISCUSSION TITLE
run_auto in completely non-interactive mode

### DISCUSSION THREAD

**Discussion by trondhindenes** [copier-org/copier#878] [Q&A]
we're considering using copier as part of a CI build to generate some dirs on-demand. It will be important to never block the execution by waiting for input, it would be better to raise an exception if for example a question is missing data etc. Is there a way to run copier's run_auto in a completely non-interactive mode and let it throw error instead of asking for input?

**Comment by yajo** [comment]
Yes, [use the `--force`](https://copier.readthedocs.io/en/stable/configuring/#force)

**Comment by trondhindenes** [comment]
I'm not sure I understand. Force seems to always overwrite without prompting. That's not what I'm after.

**Reply by yajo** [reply]
It's that and it also doesn't ask for answers.
